doc/users/next_whats_new/colorbar_has_location_argument.rst

```diff
@@ -0,0 +1,24 @@
+``colorbar`` now has a ``location`` keyword argument
+====================================================
+
+The ``colorbar`` method now supports a ``location`` keyword argument
+to more easily position the color bar. This is useful when providing
+your own inset axes using the ``cax`` keyword argument and behaves
```

Comment:
this sounds like this kwarg only works with a passed in cax & if that's not the case than please also add an example for the not-cax case